src/dialogs/EditLinkDialog.ts

```diff
@@ -0,0 +1,57 @@
+import LinkColumn from '../model/LinkColumn';
+import Column from '../model/Column';
+import ADialog from '../ui_dialogs';
+
+export default class EditLinkDialog extends ADialog {
+
+  private readonly templates;
+  private readonly idPrefix;
+
+  constructor(column: LinkColumn, $header: d3.Selection<Column>, templates: string[] = [], idPrefix: string, title: string = 'Edit Link ($ as Placeholder)') {
+    super(column, $header, title);
+
+    this.templates = templates;
+    this.idPrefix = idPrefix;
+
+    this.openDialog();
```

Comment:
I would avoid opening the dialog directly within the constructor. At least that is something I wouldn't expect, if I'm just creating the dialog.